pkg/webhook/testdata/cronjob_types.go

```diff
@@ -13,7 +13,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-//go:generate ../../../.run-controller-gen.sh webhook paths=. output:dir=.
+//go:generate ../../../.run-controller-gen.sh webhook:webhookVersions={v1beta1,v1} paths=. output:dir=.
```

Comment:
Since v1 is now the default:
```suggestion
//go:generate ../../../.run-controller-gen.sh webhook:webhookVersions={v1,v1beta1} paths=. output:dir=.
```

Reply:
Fixed